ckanext/iati_generator/blueprint/iati.py

```diff
@@ -1,6 +1,8 @@
-from flask import Blueprint, render_template
+from flask import Blueprint, render_template, request, redirect, url_for, flash, send_from_directory
```

Comment:
Replace ´render_template´ with ´ckan.lib.base.render´.
CKAN in general override flask. In this case, CKAN adds a cache layer.
Review all flask imports